docs/usage.rst

```diff
@@ -132,6 +132,23 @@ save or delete - for this, you can use the :code:`ElasticsearchTypeMixin` built-
 
 Awesome - Django's magic is applied.
 
+Notes
+=====
+
+* Prior to version 2.2.0 of this package, only models with numerical primary keys could be indexed properly due to the
+  way the :code:`queryset_iterator()` utility function was implemented. This has been changed and the primary key no longer
+  matters.
+
+  Ordering the bulk queryset is important due to the fact that records may have been added during the indexing process
+  (indexing data can take a long time); if the results are ordered properly, the indexing process will catch the most
+  recent records. For most cases, the default bulk ordering of :code:`pk` will suffice (Django's default primary key field is
+  an auto-incrementing integer).
+
+  If a model has PK using a :code:`UUIDField` however, things change: UUIDs are randomly generated, so ordering by a
+  :code:`UUIDField` PK will most likely result in newly created items being missed in the indexin process. Overriding the
+  :code:`ElasticsearchTypeMixin` class method :code:`get_bulk_ordering()` addresses this issue - set it to order by a
+  :code:`DateTimeField` on the model.
```

Comment:
@yakky I thought about adding that in... I might still in future.

@zorba-the-geek Great question, to which I have a (I think) great answer! 

You have very limited ability to 'update' or otherwise change the mapping for a document type in an Elasticsearch index.  You can add new fields/properties, but apart from some very specific exceptions, you cannot change existing fields.

Assuming you want to index documents (Django model instances from a queryset) into `myindex`. The way this app works, when you run `python manage.py es_manage --rebuild`, it: 
* creates a NEW index with a timestamp (`myindex-20170718-145616`, that sort of thing) 
* sets all the document type mappings specific to the 'myindex' into the newly created index
* it starts to index the queryset documents into the new timestamp index
  * **NOTE: `myindex` is just an index 'alias', which points to the previous timestamped index, so if you delete/change existing records or create new records while the index rebuild is happening, the changes will go into the previous timestamped index via the alias `myindex`.**
* once the index rebuild process is complete, the index alias for `myindex` is switched to point to the new `myindex-20170718-145616` timestamped index we just rebuilt.

I hope that explains it?